firmware/stackchan/transcriptions/stt-whisper.ts

```diff
@@ -0,0 +1,61 @@
+import { fetch } from 'fetch'
+import Headers from 'headers'
+import UUID from 'uuid'
+import type { Maybe } from 'stackchan-util'
+
+export type STTProperty = {
+  apiKey: string
+  model?: string
+  language?: string
+}
+
+export default class STT {
+  apiKey: string
+  model: string
+  language: string
+
+  constructor(props: STTProperty) {
+    this.apiKey = props.apiKey
+    this.model = props.model ?? 'whisper-1'
+    this.language = props.language ?? 'ja'
+  }
+  async transcription(audio: ArrayBuffer | HostBuffer): Promise<Maybe<string>> {
```

Comment:
- 動詞形（transcribe）にする
- 引数名は、 `audioOut`等との混同を避けるために `audioBuffer`または単に `buffer` などとする

```suggestion
  async transcribe(audioBuffer: ArrayBuffer | HostBuffer): Promise<Maybe<string>> {
```